src/schema/common.ts

```diff
@@ -217,16 +212,16 @@ export const fixedIdsPageGenerator = <
   nodeToCursor: (page, args, node, i, queryParams): string =>
     offsetToCursor(
       page.offset +
-        queryParams.data.findIndex(([postId]) => postId === node.id),
+        queryParams!.data.findIndex(([postId]) => postId === node.id),
```

Comment:
Using `!` in this file because code so far expected no `undefined` so if anything was to break it should break here as well instead of guarding and then returning wrong data